client/src/components/App.jsx

```diff
@@ -6,14 +6,34 @@ import axios from 'axios';
 
 const App = () => {
 
+  const getFromDB = () => {
```

Comment:
maybe something like getCampsiteById ? referencing DB from client is an antipattern.